### ISSUE TITLE
escaping string values

### ISSUE THREAD

**Issue by james-hu** [luyilin/json-format-highlight#3]
Hi, Luyilin,

This is very handy, just a problem when string values contain HTML tags, they are currently not escaped.

Below is how I copy-and-fixed it in my own work:

```javascript               
                    var color = colors.numberColor;
                    var styles = '';
                    if (/^"/.test(match)) {
                        if (/:$/.test(match)) {
                            color = colors.keyColor;
                        } else {
                            color = colors.stringColor;
                            match = escapeHtml(match);
                            styles = ';word-wrap:break-word;white-space:pre-wrap';
                        }
                    } else {
                        color = /true/.test(match) ? colors.trueColor : /false/.test(match) ? colors.falseColor : /null/.test(match) ? colors.nullColor : color;
                    }
                    return ("<span style=\"color: " + color + styles + "\">" + match + "</span>");
 ```
Hope you can fix the problem so that I can just use your library directly.

Thanks!

**Comment by luyilin** [comment]
Maybe someone needs unescaped string, so I won't add this feature, feel free to use it in your own project 😉

**Comment by james-hu** [comment]
![image](https://user-images.githubusercontent.com/8565700/67357496-1f01ca80-f5a9-11e9-88d5-15fa5b666326.png)
 Are you sure this is most people want?

Instead of this?
![image](https://user-images.githubusercontent.com/8565700/67357590-66885680-f5a9-11e9-9477-c195e4d1f214.png)

**Comment by luyilin** [comment]
You are right 😅, you can create a pr for this~

**Comment by james-hu** [comment]
PR created

**Comment by luyilin** [comment]
released on v1.0.2
